README.md

```diff
@@ -1 +1,60 @@
-# help-editor-plugin
+# Help Editor Plugin
+
+Help editor plugin allows you to add additional custom help above default help on job configuration pages.
+
+## Table of contents
+1. [Quick Start](#quick-start)
+2. [Current status](#current-status)
+3. [Plugin Structure](#plugin-structure)
+
+## Quick Start
+
+After downloading help editor plugin you will need to make only few easy steps to create your custom help information:
+
+1. Open job configuration page with all settings
+2. Click blue question mark icon that would expand help information
+3. Click 'Edit' button in the new field above custom help
+4. Enter your information (text supports HTML markup)
+5. Click save button
+
+Then you will be able to see your custom information, each time you open your help.
+
+You can see this process illustrated on the GIF below:
+
+![Demonstrating GIF](https://files.gitter.im/samatdav/DS4T/PluginDemo.gif)
```

Comment:
Can this also show the result after the edit is made when you reload the page the edit is still there...